src/h5web/dataset-visualizer/DatasetVisualizer.tsx

```diff
@@ -25,22 +25,20 @@ function DatasetVisualizer(props: Props): ReactElement {
   const { dataset } = props;
 
   const supportedVis = useSupportedVis(dataset);
-  const [state, mergeState] = useSetState<State>({});
+  const [state, mergeState] = useSetState<State>({ activeVis: Vis.Raw });
```

Comment:
I would put a comment here to tell that you are "allowed" to do that because `Vis.Raw` supports all datasets (see `visualisation/index.ts`)